packages/react-charting/src/components/PieChart/__snapshots__/PieChart.test.tsx.snap

```diff
@@ -13,81 +13,159 @@ exports[`PieChart snapShot testing renders PieChart correctly 1`] = `
         width: 670px;
       }
 >
-  <svg
-    height={350}
-    width={600}
+  <div
+    className=
+        ms-FocusZone
+        &:focus {
+          outline: none;
+        }
+    data-focuszone-id="FocusZone0"
+    onFocus={[Function]}
+    onKeyDown={[Function]}
+    onMouseDownCapture={[Function]}
   >
-    <g
-      transform="translate(300, 175)"
+    <svg
+      height={350}
+      width={600}
     >
       <g
-        className="arc"
+        transform="translate(300, 175)"
       >
-        <path
+        <g
+          aria-label="A-50"
           className=
```

Comment:
I see that className has disappeared in the snapshot. Could we get it back.

Reply:
I think we're refering 
      <g
        className="arc"
        transform="translate(300, 175)"
      >
      
I've added the class back to this snapshot, if it's some other place please let me know.

Reply:
I was referring to classname = arc. That is still missing.